index.html

```diff
@@ -54,7 +54,170 @@ <h2 id="name_title">I’m Chima Glad to see you!</h2>
+                <p class="mini_title">LET’S CONNECT</p>
+                <nav class="about_social_links">
+                    <ul class="social_icons">
+                        <li id="twitter_icon">
+                            <a href="#">
+                                <img src="assets/images/twitter.svg" alt="Twitter" class="icon_image">
+                            </a>                            
+                        </li>
+                        <li id="linkedin_icon">
+                            <a href="#">
+                                <img src="assets/images/linkedin.svg" alt="Linkedin" class="icon_image">
+                            </a>                            
+                        </li>
+                        <li id="medium_icon">
+                            <a href="#">
+                                <img src="assets/images/medium.svg" alt="Medium" class="icon_image">
+                            </a>                            
+                        </li>
+                        <li id="git_icon">
+                            <a href="#">
+                                <img src="assets/images/git.svg" alt="Git" class="icon_image">
+                            </a>                           
+                        </li>
+                        <li id="angelist_icon">
+                            <a href="#">
+                                <img src="assets/images/angelist.svg" alt="Angelist" class="icon_image">
+                            </a>                            
+                        </li>
+                    </ul>
+                </nav>
+                <a href="#" class="cta_btn"><span class="cta_text">Get my resume</span></a>
+                <div id="profile_breakdown">
+                    <ul class="breakdown_list">
+                        <li class="breakdown_list_item active">
+                            <span class="accordio_cta">Languages
+                                <img src="assets/images/Union.png" alt="Collapse">
+                            </span>
+                            <ul class="breakdown_list_children">
+                                <li class="breakdown_list_children_item"><img src="assets/images/js.png" alt="Javascript"></li>
+                                <li class="breakdown_list_children_item"><img src="assets/images/html.png" alt="Html"></li>
+                                <li class="breakdown_list_children_item"><img src="assets/images/css.png" alt="Javascript"></li>
+                            </ul>
+                        </li>
+                        <li class="breakdown_list_item"><span class="accordio_cta">Frameworks <img src="assets/images/chevron.png" alt="Expand"></span></li>
```

Comment:
<img width="762" alt="8" src="https://user-images.githubusercontent.com/77843050/212476859-957d0ddb-7518-4cd3-bd50-243153cb8e11.png">

- [ ] The image said it all, kindly add the correct spaces after the `get my resume` button
- [ ] And adjust the icons as pointed, take a look at the original design as reference